sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Disallow space character when validating tenant id and scopes as input for `AzureCliCredential`.
```

Comment:
Arguably this is a breaking change.

Reply:
Theoretically yes, but there is no supported valid scenario where that would break a customer since no valid tenant id/single scope allows it. Hence, I didn't put it as breaking.